modules/oxide/src/less/skins/content/dark/content.less

```diff
@@ -25,12 +25,64 @@ table {
   border-collapse: collapse;
 }
 
-table th,
-table td {
-  border: 1px solid #6d737b;
+/* Apply a default padding if legacy cellpading attribute is missing */
```

Comment:
This seems to be duplicated 4 times, so this should probably be moved to another file and then imported.

Reply:
Also once moved, could we add a comment about this being a workaround, since the dialogs don't handle this complexity very well?

Reply:
This is the content styles that integrators should adapt to their particular use-case and it showcases what it takes to make the dialogs work somewhat with predefined table styles (using totally unstyled tables expecting end-users to style them from scratch is not something people (should) do out of perhaps using tables for layout.). 

I'm replicating the css here because I don't want to complicate these files with imports, readability trumps efficiency. As a side note, they were ment to be plain css files, not less files but I simply forgot that back in the day and haven't bothered fixing it and at this time it's probably a breaking change anyway.

What is it that the dialogs won't handle that it did before?

Reply:
The source is for us and it's meant to be maintainable. This is not maintainable and will only cause headaches down the line... we have distribution files and docs for the purpose you mentioned, so customers should be using that, not looking at these files.

As an example, see the following:
- https://www.tiny.cloud/docs/configure/content-appearance/#content_css
- https://www.tiny.cloud/docs/general-configuration-guide/boilerplate-content-css/

You can see in the first, we tell customers to look at the compiled dist repo for these files and that makes a lot more sense. So, no this file should not have duplicate code.

Reply:
No, to properly make skins and content css:es [you're supposed to dig into the source and build TinyMCE](https://github.com/tinymce/tinymce/issues/5616). The theme folder is primarily for internal use, the skins folder are for the integrators.

Reply:
While being open source means that people are able to dig into the source when the docs are unclear, that's not the workflow we want to optimise for. Having this duplication in the source also increases the chances that later down the line the files will go out of sync, and the person reading / copying the css will come out with the wrong version of the code.

Reply:
You've mentioned the intention is for the integators to edit the skins/styles, that is an advanced concept, and will require the integrator to work with less variables anyway so it really should all be kept the same. In addition to that, they still need to compile and use the **dist/build** copy anyways, so it makes more sense to keep it all consistent both from an editing and maintainability perspective. If they want something to copy/paste, that's where the other resources I've linked already come into play (eg for basic users)

Anyways, I talked to the team on this and they were all in agreement that this should **not** be a single file that's duplicated, so my request changes comment still stands sorry.

Reply:
There are more considerations

* Introducing a folder inside skins directory that is not a skin is confusing. And the skins folder are architected to be self contained. Remember they were originally ment to contain css files.
* Rewriting the build scripts to exclude the folder and it's files is out of scope for this fix.
* Introducing an ad-hoc variable to manage that single different border color and a) explain that this one isan exception to the 300 other ones and not part of the Style API, b) explain it has to be positioned below the the import to work is not good. The other way would be to use mixins I'm not planning on introducing mixins for this only, we're not using them anywhere else in oxide.
* Much of the other CSS is duplicated already so this PR doesn't introduce anything that wasn't there before. So for consistency I don't want to just break tables out. It might be time to revisit the content css architecture but this is not the time and place to do that. 
* Dry CSS/LESS/SCSS is difficult to read and understand.

I'm getting where you come from and as said previously, it might soon be time to revisit the skin architecture but not here and now. I'm more than happy to revisit this in the future.

@ashfordneil That might be true for TinyMCE javascript, but for building skins, you truly are ment to dig into the source, modify the variables and build your skin. You also identified the reason we don't offer custom skins on cloud, because the skins will become out of date.

Reply:
Note: resolved via Slack calls and conversations.